src/main/java/com/han95210/codesquad/monsterRace/step2/MonsterRace.java

```diff
@@ -0,0 +1,68 @@
+package com.han95210.codesquad.monsterRace.step2;
+
+import java.util.Scanner;
+
+public class MonsterRace {
+
+    private Scanner scanner;
+
+    private int getNumberOfMonster() {
+        scanner = new Scanner(System.in);
+
+        // 몬스터 수 정하기
+        System.out.println("몬스터는 모두 몇 마리인가요?");
+        return scanner.nextInt();
+    }
+
+    // 기능 1개 : 시도할 횟수 입력 받기
+    private int getNumberAttempts() {
+        scanner = new Scanner(System.in);
+
+        // 시도 횟수 정하기
+        System.out.println("시도할 횟수는 몇 회 인가요?");
+        return scanner.nextInt();
+    }
+
+    private static String getRandomDash() {
+        String dashStr = "";
+        int randomNum = (int) (Math.random() * 10);
+        if (randomNum >= 4) {
```

Comment:
이런 4 같은 값은 상수를 활용하시는게 어떨까요?

4라는 Integer 값을 봤을 때 이 값이 어떤 목적으로 정의되었는지 이해하기 힘들 수 있어요.

```java
private static final Integer FORWARD_NUMBER = 4;
```
```suggestion
if (randomNum >= FORWARD_NUMBER) {
```